src/data/content/quality.ts

```diff
@@ -1,12 +1,200 @@
+          },
+          {
+            name: "",
+            description: "4. Perform one or more user actions on the element. ",
+          },
+          {
+            name: "",
+            description:
+              "5. Preload the expected output/browser response to the action.",
+          },
+          {
+            name: "",
+            description: "6. Run test.",
+          },
+          {
+            name: "",
+            description:
+              "7. Record results and compare results from them to the expected output.",
+          },
+        ],
+      },
+      {
+        name: "Benefits of Selenium WebDriver",
+        intro: "",
+        data: [
+          {
+            name: "",
+            description: `<ul><li>It is one of the most popular Open-Source tools and is easy to get started with for testing web-based applications. It also allows you to perform <a href="https://www.selenium.dev/documentation/webdriver/getting_started/install_drivers/">cross browsers testing</a>.</li></ul> `,
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Supports multiple operating systems like Windows, Mac, Linux, Unix, etc.</li></ul>",
+          },
+          {
+            name: "",
+            description: `<ul><li>It provides compatibility with a range of languages including Python, Java, JavaScript, Ruby, etc.</li></ul>`,
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Provides support for modern browsers like Chrome, Firefox, Edge, Opera, Safari, and Internet Explorer.</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>Selenium WebDriver completes the execution of test scripts faster when compared to other tools</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>More Concise API (Application Programming interface) than Selenium RC’s</li></ul>",
+          },
+          {
+            name: "",
+            description:
+              "<ul><li>It also provides compatibility with iPhoneDriver, HtmlUnitDriver, and AndroidDriver</li></ul>",
+          },
+        ],
+      },
+      {
+        name: "Web Element",
+        intro: "",
+        data: [
+          {
+            name: "Web Element",
+            description: `Selenium uses Web-Element which represents a DOM element to locate an element on the page. Web Elements can be found by searching from the document root using a WebDriver instance, or by searching under another Web Element. In order to write effective tests, one must be familiar with how to find the Web Elements which are based on different properties like ID, Name, Class, XPath, CSS Selectors, link Text, etc located on <a href="https://www.selenium.dev/documentation/webdriver/elements/">Web element.</a>`,
+          },
+        ],
+      },
+      {
+        name: "Locating Elements",
```

Comment:
repeating titles. please remove this. 
![Screenshot 2022-06-07 at 16 22 48](https://user-images.githubusercontent.com/91062034/172419033-19ae6630-4676-43c5-a9c9-e6816f156c04.png)